build/preferences.js

```diff
@@ -177,6 +177,10 @@ PreferencesBuilder.prototype.preparePref = function() {
   if (this.config.PERF_LOGGING === '1') {
     this.userPrefs['dom.performance.enable_user_timing_logging'] = true;
   }
+  // Turn on Service Workers for the Raptor
+  if (this.config.RAPTOR === '1') {
```

Comment:
It's temporary, I'm not sure where we should set that. But we need this pref for the app that use Service Workers. Just few ideas that come to my mind:
* Leverage existing ```NGA_SERVICE_WORKERS``` here;
* Temporary (until SW are enabled by default) add ```'dom.serviceWorkers.enabled'``` to [settings.js](http://mxr.mozilla.org/mozilla-central/source/b2g/chrome/content/settings.js#661) so that we can turn it on via ```distros/raptor/settings.json```.

What do you think?